packages/hash/frontend/src/shared/ui/menu-checkbox-item.tsx

```diff
@@ -0,0 +1,25 @@
+import { Checkbox, ListItemIcon } from "@mui/material";
+import { ReactNode, forwardRef, ForwardRefRenderFunction } from "react";
+import { MenuItem, MenuItemProps } from "./menu-item";
+
+export type MenuCheckboxItemProps = {
+  children?: ReactNode;
+} & MenuItemProps;
```

Comment:
I think you want to omit `noSelectBackground` from the props here, as the point of this component is to hardcode it in

```suggestion
} & Omit<MenuItemProps, "noSelectBackground">;
```

Doing this will highlight that in a few places in this PR, you've left `noSelectBackground` in for uses of `MenuCheckboxItem`, even though it doesn't do anything as it's set for you here.

Reply:
True. Thanks for pointing this out. Updated in [d5166d8](https://github.com/hashintel/hash/pull/547/commits/d5166d8991a62ec45f9a9830be59cc1e3c40fece)